plugins/labhub.py

```diff
@@ -89,40 +100,28 @@ def invite_cmd(self, msg, match):
         team = 'newcomers' if match.group(2) is None else match.group(2)
         team = team.lower()
 
-        is_developer = self.TEAMS[self.GH_ORG_NAME +
-                                  ' developers'].is_member(inviter)
-        is_maintainer = self.TEAMS[self.GH_ORG_NAME +
-                                   ' maintainers'].is_member(inviter)
+        is_developer = self.is_team_member(inviter, 'developers')
+        is_maintainer = self.is_team_member(inviter, 'maintainers')
 
         self.log.info('{} invited {} to {}'.format(inviter, invitee, team))
 
-        valid_teams = ['newcomers', 'developers', 'maintainers']
```

Comment:
`valid_teams = elf.team_mapping()` will require less changes to existing lines, and be more efficient.